.circleci/config.yml

```diff
@@ -825,6 +852,43 @@ jobs:
             instance_id=$(AWS_ACCESS_KEY_ID=${TEST_AWS_ACCESS_KEY_ID} AWS_SECRET_ACCESS_KEY=${TEST_AWS_SECRET_ACCESS_KEY} aws --region us-west-2 ec2 describe-instances --filters "Name=tag:Name,Values=$name" --query 'Reservations[].Instances[].InstanceId' --output text)
             AWS_ACCESS_KEY_ID=${TEST_AWS_ACCESS_KEY_ID} AWS_SECRET_ACCESS_KEY=${TEST_AWS_SECRET_ACCESS_KEY} aws --region us-west-2 ec2 terminate-instances --instance-ids $instance_id
 
+  build_and_sign_job:
+    machine:
+      image: ubuntu-2004:202010-01
+    resource_class: large
+    environment:
+      TMPDIR: /mnt/ramdisk
+    working_directory: /home/circleci/go/src/github.com/influxdata/influxdb
+    steps:
+      - checkout
+      - run:
+          name: checkout by tag
+          command: |
+            git fetch --all --tags
+            git checkout tags/<< pipeline.parameters.build_and_sign_tag >> -b << pipeline.parameters.build_and_sign_tag >>
+      - run:
+          name: Create RAM disk
```

Comment:
copying the code from the release job make me very nervous - can we re-use existing job definitions here, maybe with some `when` clauses or splitting the existing jobs?

Reply:
probably, but I felt like all the conditions made the logic much harder to track. I'll add another commit and you can see what you think.

Reply:
Turns out it was very easy to eliminate the extra job :). Definitely an improvement